doc/source/dev/toolchain.rst

```diff
@@ -188,20 +225,20 @@ CPython) has been recent enough to support even C++17.
 Since the official builds (see above) use a pretty recent version of LLVM,
 the bottleneck for C++ support is therefore the oldest supported GCC version,
 where SciPy has been constrained mainly by the version in the oldest supported
-manylinux versions & images [14]_.
+manylinux versions & images [16]_.
 
 At the end of 2021 (with the final removal of ``manylinux1`` wheels), SciPy
 now has a minimum GCC requirement of GCC 6.3, which has full C++14 support
-[15]_. This corresponds to the lowest present GCC version in relevant manylinux
+[17]_. This corresponds to the lowest present GCC version in relevant manylinux
 versions - somewhat surprisingly, it is not the oldest remaining
 ``manylinux2010`` that is the most restrictive (due to the ABI-compatible
 "RHEL Dev Toolset" backports, it has GCC 8.3), but actually ``manylinux_2_24``
-that only comes with GCC 6.3 [16]_.
+that only comes with GCC 6.3 [18]_.
```

Comment:
Since I intend this PR to be backportable to 1.9, I didn't touch the EOL of manylinux_2_24 and its implications. We can discuss this in a future PR for (hopefully) 1.10